src/compaction.h

```diff
@@ -16,9 +16,13 @@
 #define AGG_MAX 2
 #define AGG_SUM 3
 #define AGG_AVG 4
-#define AGG_COUNT 5
-#define AGG_FIRST 6
-#define AGG_LAST 7
+#define AGG_STD_P 5
```

Comment:
Why not add them in the end?

Reply:
My thought was, standard deviation belongs with average so I put them together.